packages/sso-menu/src/SSOMenu.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@leafygreen-ui/button';
+import Icon from '@leafygreen-ui/icon';
+import { Menu, MenuItem, MenuGroup } from '@leafygreen-ui/menu';
+import { uiColors } from '@leafygreen-ui/palette';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+const menuButtonStyle = css`
+  height: 29px;
+  padding: 2px 14px;
+  border: 1px solid ${uiColors.gray.base};
+  border-radius: 14.5px;
+  cursor: pointer;
+  transition: background 200ms ease-in-out;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: ${uiColors.gray.dark1};
+  font-size: 12px;
+  line-height: 15px;
+  position: relative;
+  &:hover {
+    background-color: ${uiColors.gray.light1};
+    color: ${uiColors.gray.dark2};
+  }
+`;
+
+const menuNameStyle = css`
+  margin-right: 2px;
+  margin-left: 2px;
+`;
+
+const activeMenuButtonStyle = css`
+  background-color: ${uiColors.gray.base};
+  color: ${uiColors.white};
+  &:hover {
+    background-color: ${uiColors.gray.base};
+    color: ${uiColors.white};
+  }
+`;
+
+const nameStyle = css`
+  font-size: 16px;
+  color: ${uiColors.gray.dark1};
+  font-weight: bolder;
+  margin-top: 4px;
+  margin-bottom: 2px;
+  margin-right: 0px;
+  margin-left: 0px;
+`;
+
+const truncate = css`
+  width: 162px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const iconStyle = css`
+  position: relative;
+  fill: ${uiColors.gray.dark1} !important;
+  transition: color 200ms ease-in-out;
+`;
+
+const accountMenuGroupStyle = css`
+  padding: 20px 20px 14px;
+`;
+
+const accountButtonStyle = css`
+  margin-top: 12px;
+  width: 100%;
+  display: inline-flex;
+  justify-content: center;
+`;
+
+const menuItemTextStyle = css`
+  font-size: 14px;
+  line-height: 16px;
+  cursor: pointer;
+  margin: 0px;
+`;
+
+const descriptionStyle = css`
+  margin: 0px;
+  display: block;
+  font-weight: normal;
+  font-size: 12px;
+  color: ${uiColors.gray.dark2};
+  text-decoration: none;
+`;
+
+const menuItemPadding = css`
+  padding-left: 20px;
+`;
+
+const logoutContainerHeight = css`
+  height: 46px;
+`;
+
+const menuItems = [
```

Comment:
How come you implemented them this way?

```js
const menuItems = [
  {
    displayName: 'Atlas',
    description: 'cloud.mongodb.com',
    href: 'https://cloud.mongodb.com',
    slug: 'atlas',
  },
  {
    displayName: 'University',
    description: 'university.mongodb.com',
    href: 'https://university.mongodb.com',
    slug: 'university',
  },
  {
    displayName: 'Cloud Support',
    description: 'support.mongodb.com',
    href: 'https://support.mongodb.com',
    slug: 'support',
  },
] as const;

<MenuGroup>
	{menuItems.map(el => (
		<MenuItem
			onClick={onProductChange}
			key={el.displayName}
			active={el.slug === activeProduct}
			href={el.href}
			description={el.description}
		>
			{el.displayName}
		</MenuItem>
	))}
</MenuGroup>
```
Rather than:
```js
<MenuGroup>
	<MenuItem
		onClick={onProductChange}
		active={activeProduct === 'atlas'}
		href='https://cloud.mongodb.com'
		description={cloud.mongodb.com}
	>
		Atlas
	</MenuItem>

	<MenuItem
		onClick={onProductChange}
		active={activeProduct === 'university'}
		href='https://university.mongodb.com'
		description={university.mongodb.com}
	>
		University
	</MenuItem>

	<MenuItem
		onClick={onProductChange}
		active={activeProduct === 'support'}
		href='https://support.mongodb.com'
		description={support.mongodb.com}
	>
		Support
	</MenuItem>
</MenuGroup>
```

Reply:
Felt more concise to me, but happy to implement either way.

Reply:
I'm wondering if this was a bit more clear without having to look at the entire file, but maybe it's fine 🤷‍♂️